crates/dojo/lang/src/inline_macros/poseidon_hash_string.rs

```diff
@@ -0,0 +1,62 @@
+use cairo_lang_defs::patcher::PatchBuilder;
+use cairo_lang_defs::plugin::{
+    InlineMacroExprPlugin, InlinePluginResult, MacroPluginMetadata, NamedPlugin, PluginDiagnostic,
+    PluginGeneratedFile,
+};
+use cairo_lang_defs::plugin_utils::unsupported_bracket_diagnostic;
+use cairo_lang_diagnostics::Severity;
+use cairo_lang_syntax::node::{ast, TypedStablePtr, TypedSyntaxNode};
+use dojo_types::naming;
+
+#[derive(Debug, Default)]
+pub struct PoseidonHashStringMacro;
+
+impl NamedPlugin for PoseidonHashStringMacro {
+    const NAME: &'static str = "poseidon_hash_string";
```

Comment:
```suggestion
    const NAME: &'static str = "poseidon_hash_string";
```
Wondering if `poseidon_hash_bytearray` would not be more consistent with the dojo utils functions already using `bytearray` instead of string.